index.d.ts

```diff
@@ -120,3 +120,8 @@ console.log(error);
 ```
 */
 export function deserializeError(errorObject: ErrorObject | unknown, options?: Options): Error;
+
+/**
+Predicate to determine whether an object looks like an error, even if it's not an instance of Error
```

Comment:
```suggestion
Predicate to determine whether an object looks like an error, even if it's not an instance of `Error`.
```

Reply:
It should also mention what properties it expects it to have to be "error-like".